test/rules/unused_struct_field/access_field.exs

```diff
@@ -0,0 +1,14 @@
+defmodule MeandroTest.MyBeh do
```

Comment:
MyBeh was for My Behaviour… You should probably pick a different name for this module.

Reply:
Yep!